Tests/TestingApps/PaywallsTester/PaywallsTester/Views/AppContentView.swift

```diff
@@ -139,16 +134,75 @@ struct AppContentView: View {
         }
     }
 
-    private var isPurchasesConfigured: Bool {
-        return self.customerInfoStream != nil
+    private func reconfigure(for mode: Configuration.Mode) {
+        configuration.reconfigure(for: mode)
```

Comment:
```suggestion
        self.configuration.reconfigure(for: mode)
```

Reply:
feels like a linter or formatter should be automating this instead of us relying on manual linting? Don't we have swiftlint enabled for this app?